plugins/plugin-apache-composer/src/test/composer1/composer-kill.ts

```diff
@@ -28,7 +28,6 @@
 // const killFailure404 = 'Cannot find live session'
 
 import { ISuite } from '@kui-shell/core/tests/lib/common'
-import * as openwhisk from '@kui-shell/plugin-openwhisk/tests/lib/openwhisk/openwhisk'
```

Comment:
openwhisk is actually used in the test content (which has been commented out). If we want to keep this commented content, maybe we should comment out this line instead of remove it.

@starpit , do we want to keep this test?

Reply:
For the other files in this PR, if the unused var is used in commented content, maybe we should also comment out the variable for consistency.

Reply:
The whole file is commented out, I think we should drop the file.

About the rest of the files, it's not easy because the linter can't help me with that. I would use the same approximation: to keep the linter happy to close this as soon as possible and to drop all the code which is commented out in a new PR.